src/components/basic-table/SecondaryBasicTable.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { TableBuilderColumn } from 'baseui/table-semantic';
+import { BasicTableProps, BasicTableRow, BasicTableColumnType } from './types';
+import { renderCell, BasicTableBuilder, BasicTableHeadCell } from './components';
+import { useBasicTableStyles } from './hooks';
+
+export const SecondaryBasicTable = ({ columns, emptyMessage, ...props }: BasicTableProps) => {
```

Comment:
I would prefer another name that more clearly indicates the difference between the two table types